fiftyone/zoo/models/manifest-torch.json

```diff
@@ -2348,9 +2348,361 @@
                     "support": true
                 }
             },
-            "tags": ["detection", "coco", "torch"],
+            "tags": ["detection", "coco", "torch", "yolo"],
             "date_added": "2023-08-22 19:22:51"
         },
+        {
+            "base_name": "yolov8n-coco-torch",
+            "base_filename": "yolov8n-coco.pt",
+            "description": "Ultralytics YOLOv8n model trained on COCO",
+            "source": "https://docs.ultralytics.com/models/yolov8/",
+            "size_bytes": 6534387,
+            "manager": {
+                "type": "fiftyone.core.models.ModelManager",
+                "config": {
+                    "url": "https://github.com/ultralytics/assets/releases/download/v8.1.0/yolov8n.pt"
+                }
+            },
+            "default_deployment_config_dict": {
+                "type": "fiftyone.utils.ultralytics.FiftyOneYOLODetectionModel",
+                "config": {}
```

Comment:
FYI actually no parameters are required here. `load_zoo_model()` automatically downloads the model and injects the path into the `model_path` of the config